.travis.yml

```diff
@@ -26,6 +26,7 @@ env:
   - HELLBENDER_TEST_INPUTS=gs://hellbender/test/resources/
   - HELLBENDER_TEST_STAGING=gs://hellbender/test/staging/
   - HELLBENDER_TEST_PROJECT=broad-dsde-dev
+  - HELLBENDER_JSON_SERVICE_ACCOUNT_KEY=servicekey.json
```

Comment:
Actually, can you just use the existing `GOOGLE_APPLICATION_CREDENTIALS` variable?